helm/idol-nifi/templates/nifi/hpa.yaml

```
@@ -12,14 +12,14 @@
 apiVersion: autoscaling/v2
 kind: HorizontalPodAutoscaler
 metadata:
-  name: idol-nifi
+  name: nifi
```

Would make sense for this to use same name as the statefulset target
```suggestion
  name: {{ .Values.name }}
```